tests/playwright/utils/pageObjects/base.page.ts

```diff
@@ -0,0 +1,72 @@
+import { expect, Locator, Page } from "@playwright/test";
+import AxeBuilder from "@axe-core/playwright";
+
+export default class BasePage {
+  public path = "/";
+
+  readonly page: Page;
+  readonly title: Locator;
+  readonly continueButton: Locator;
+  readonly previousButton: Locator;
+  readonly myAccountButton: Locator;
+  readonly accountMenu: Locator;
+  readonly manageAccountButton: Locator;
+  readonly logoutButton: Locator;
+
+  constructor(page: Page) {
+    this.page = page;
+    this.title = page.getByRole("heading", {
+      name: "Managed Care Reporting",
+    });
+    this.continueButton = page.getByRole("button", { name: "Continue" });
+    this.previousButton = page.getByRole("button", { name: "Previous" });
+    this.myAccountButton = page.getByRole("button", { name: "my account" });
```

Comment:
Nit: This button is called in login utils with `page.getByRole("button", { name: "My Account" });`. I think in that case it's using the button text, while here is using the `aria-label="my account"`. It seems like the label isn't needed?